src/Tasks/Microsoft.NET.Build.Tasks/build/platforms/Xamarin.TVOS.targets

```diff
@@ -0,0 +1,22 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(_ShortFrameworkIdentifier)' == 'xamarintvos' or '$(_ShortFrameworkIdentifier)' == 'xamarin.tvos' ">
+    <TargetFrameworkVersion Condition="'$(TargetFrameworkVersion)' == ''">v1.0</TargetFrameworkVersion>
+    <IPhoneResourcePrefix Condition="'$(IPhoneResourcePrefix)' == ''">Resources</IPhoneResourcePrefix>
+    <!-- Default to full until Xamarin supports PPDB's -->
+    <DebugType>full</DebugType>
```

Comment:
I certainly hope this can be changed later -- I put it there right now because it's effectively required. The Xamarin toolchain will error out with ppdb's right now.